profiles/kentik_snmp/raritan/dominion.yml

```diff
@@ -0,0 +1,66 @@
+# https://mibs.observium.org/mib/RemoteKVMDevice-MIB/
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.3.2.10    # Raritan Dominion SXII DSX2-32
+  - 1.3.6.1.4.1.13742.*    # Raritan Devices
+
+provider: kentik-switch
+
+metrics:
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: remoteKVMDeviceGet
+      OID: 1.3.6.1.4.1.13742.3.1
+    symbols:
+      # Percent current memory usage
+      - name: systemUsageMemory
+        OID: 1.3.6.1.4.1.13742.3.1.1
+      # Percent current processor usage
+      - name: systemUsageCPU
+        OID: 1.3.6.1.4.1.13742.3.1.2
+        tag: CPU
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: systemPowerSupplyTable
+      OID: 1.3.6.1.4.1.13742.3.1.3
+    symbols:
+      # Power supply data
+      - name: systemPowerSupplyEntry
+        OID: 1.3.6.1.4.1.13742.3.1.3.1
+      # Is the system power supply on (true)
+      - name: systemPowerSupplyPowerOn
+        OID: 1.3.6.1.4.1.13742.3.1.3.1.2
+        enum:
+          false: 0
+          true: 1
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.13742.3.1.3.1.1
+          name: systemPowerSupplyNumber
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: portDataTable
+      OID: 1.3.6.1.4.1.13742.3.1.4
+    symbols:
+      # Physical port number
```

Comment:
don't need this as it is just a duplicate of the Index for this table.
Should remove lines 49-51

Reply:
Removed